crates/starknet-devnet-server/src/api/http/endpoints/dump_load.rs

```diff
@@ -25,24 +25,37 @@ pub(crate) async fn dump_impl(api: &Api, path: DumpPath) -> HttpApiResult<()> {
     match path.path {
         None => {
             // path not present
-            starknet
-                .dump_events()
-                .map_err(|err| HttpApiError::DumpError { msg: err.to_string() })?;
-            Ok(())
+            match starknet.config.dump_path.clone() {
```

Comment:
Forgot to mention that this line can match a reference, and later the cloning can be done in L32.
```suggestion
            match &starknet.config.dump_path{
```